### DISCUSSION TITLE
Update Order history page to paste component

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#927] [Q&A] [answered]
We like to apply Default theme and update Super SIM order history page to Paste components.

Guidance on the layout

1) the component usage
2) What component to use for showing the shipment progress

Invision Design is shared in the - DSys office hour signup sheet.

**Comment by kate-porter** [comment, marked as answer]
Hey @ishah-twilio,

Overall component usage on this page looks good. Thanks for using the Default theme! Just a couple things:
- It looks like "manage SIMs" takes the user to a new page, which means [it should be an Anchor instead of a Button](https://paste.twilio.design/components/button#button-vs-anchor-link). If you tried using an Anchor instead of a Button and encountered usability issues because of an impact to the visual hierarchy, this _can_ be an Anchor styled as a Button, but please consider adding a right arrow icon to it to indicate to the user that they will be navigated to a new page (you can see an example of this recommendation in the [Create pattern](https://paste.twilio.design/patterns/create#anchor)).
- The text under the "manage SIMs" button (starting with, "ordered more than...") looks like it's using `color-text-weaker`, which we reserve for disabled elements, as it does not meet a 4.5:1 contrast ratio which is an accessibility requirement for any non-disabled elements. Can you update that to `color-text-weak`?

There is no single component that you can use to show the shipment progress in your design, but you should be able to partner with your engineers to build that using Box, Text, and the Success icon.

**Reply by ishah-twilio** [reply]
Thank you. I have incorporated your feedback in design.

1) converted Manage SIMs as an anchor with icon
2) change the supporting text to color-text-weak.

Thank you.
